pyClient/zeth/zeth_address.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) 2015-2020 Clearmatics Technologies Ltd
+#
+# SPDX-License-Identifier: LGPL-3.0+
+
+from __future__ import annotations
+from zeth.ownership import OwnershipPublicKey, OwnershipSecretKey, \
+    OwnershipKeyPair, ownership_key_as_hex, gen_ownership_keypair, \
+    ownership_public_key_from_hex, ownership_secret_key_from_hex
+from zeth.encryption import \
+    EncryptionKeyPair, EncryptionPublicKey, EncryptionSecretKey, \
+    generate_encryption_keypair, encryption_public_key_as_hex, \
+    encryption_public_key_from_hex, encryption_secret_key_as_hex, \
+    encryption_secret_key_from_hex
+import json
+from typing import Dict, Any
+
+
+class ZethAddressPub:
+    """
+    Public half of a zethAddress.  addr_pk = (a_pk and k_pk)
+    """
+    def __init__(self, a_pk: OwnershipPublicKey, k_pk: EncryptionPublicKey):
+        self.a_pk: OwnershipPublicKey = a_pk
+        self.k_pk: EncryptionPublicKey = k_pk
+
+    def __str__(self) -> str:
+        """
+        Write the address as "<ownership-key-hex>:<encryption_key_hex>".
+        (Technically the ":" is not required, since the first key is written
+        with fixed length, but a separator provides some limited sanity
+        checking).
+        """
+        a_pk_hex = ownership_key_as_hex(self.a_pk)
+        k_pk_hex = encryption_public_key_as_hex(self.k_pk)
+        return f"{a_pk_hex}:{k_pk_hex}"
+
+    @staticmethod
+    def parse(key_hex: str) -> ZethAddressPub:
+        owner_enc = key_hex.split(":")
+        if len(owner_enc) != 2:
+            raise Exception("invalid JoinSplitPublicKey format")
+        a_pk = ownership_public_key_from_hex(owner_enc[0])
+        k_pk = encryption_public_key_from_hex(owner_enc[1])
+        return ZethAddressPub(a_pk, k_pk)
+
+
+class ZethAddressPriv:
+    """
+    Secret addr_sk, consisting of a_sk and k_sk
```

Comment:
Let's change to `Secret half of a zethAddress.  addr_sk = (a_sk and k_sk)` to be consistent with the comment you made above in `ZethAddressPub`

Reply:
Changed the comment.